src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/webdriver/JavaScriptUtils.kt

```diff
@@ -5,11 +5,10 @@ import org.openqa.selenium.WebDriver
 
 class JavaScriptUtils {
     companion object {
-
         @JvmStatic
-        fun <T> executeScript(driver: WebDriver, js: String): T {
+        fun <T> executeScript(driver: WebDriver, js: String, vararg args: Any): T {
```

Comment:
It's ok to change this API, because it was not released yet:
`git tag --contains ba5c8224691e748c6cf74a697a651c4002724fad`